src/app/core/data/base-response-parsing.service.ts

```diff
@@ -51,7 +51,7 @@ export abstract class BaseResponseParsingService {
         return this.processArray(data, request);
       } else if (isRestDataObject(data)) {
         const object = this.deserialize(data);
-        if (isNotEmpty(data._embedded)) {
+        if (isNotEmpty(data._embedded) && hasValue(object)) {
```

Comment:
Perhaps it makes more sense to move this `hasValue(object)` down to the `if` on line 60. That way even if the main object in a response can't be parsed, its embeds will still be processed.